### DISCUSSION TITLE
Allow fullscreen by MAUI.Blazor calls

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#22244] [Ideas]
### Description

I desire a support to the scenario of requesting fullscreen using javascript in MAUI.Blazor

### Public API Changes

: .

### Intended Use-Case

For making the app fullscreen by using javascript in Blazor

**Comment by jfversluis** [comment]
Could you please provide some more detail? What is the API in JavaScript to do this? Does this not do this today? Did you try anything to make it work? Does it have to be JavaScript? Can it be .NET APIs?

**Comment by Xyncgas** [comment]
In Maui.Blazor one can press F12 to open developer console as you can in a web app

Input ```await document.activeElement.requestFullscreen()``` nothing happens

It makes browser fullscreen according to (https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen)

I request it can be for me to use

the result for calling this inside javascript should let the _blazor document_ occupy the entire computer sceen without showing application border / window controls (minimize, maximize, closing) on the windows platform
